src/Makefile.am

```diff
@@ -1,15 +1,9 @@
 sbin_PROGRAMS = hulda
 
-INCLUDES = \
-	`pkg-config --cflags libosso` \
-	`pkg-config --cflags glib-2.0` \
-	`pkg-config --cflags gconf-2.0` \
-	`pkg-config --cflags dbus-1` \
-	`pkg-config --cflags dbus-glib-1` \
-	-DLOCALEDIR=\"`pkg-config --variable=localedir osso-af-settings`\"
+INCLUDES = -DLOCALEDIR=\"$(LOCALEDIR)\"
```

Comment:
Correct value for this line would be:
AM_CPPFLAGS = -DLOCALEDIR=\"$(LOCALEDIR)\"